test/metrix/adapters/log_test.exs

```diff
@@ -1,22 +1,27 @@
-defmodule ElixirTools.Metrix.Adapters.LogTest do
+defmodule Metrix.Adapters.LogTest do
   use ExUnit.Case, async: false
 
   import ExUnit.CaptureLog
+  import ElixirTools.MetrixHelper
 
+  alias ElixirTools.Metrix
   alias ElixirTools.Metrix.Adapters.Log
```

Comment:
This does not match the new module name `Metrix.Adapters.LogTest` now. It doesn't need to but I'm just interested in why are we renaming the modules?

Reply:
ups, renaming issue